extensions/mssql/src/mssql.d.ts

```diff
@@ -495,3 +497,43 @@ export interface ISqlAssessmentService {
 	getAssessmentItems(ownerUri: string, targetType: azdata.sqlAssessment.SqlAssessmentTargetType): Promise<azdata.SqlAssessmentResult>;
 	generateAssessmentScript(items: azdata.SqlAssessmentResultItem[], targetServerName: string, targetDatabaseName: string, taskExecutionMode: azdata.TaskExecutionMode): Promise<azdata.ResultStatus>;
 }
+
+
+/**
+ * Sql Migration
+ */
+
+// SqlMigration interfaces  -----------------------------------------------------------------------
+
+export interface SqlMigrationImpactedObjectInfo {
+	name: string;
+	impactDetail: string;
+	objectType: string;
+}
+
+export interface SqlMigrationAssessmentResultItem {
+	rulesetVersion: string;
```

Comment:
Are all of these always defined? Is there a chance any of them could be undefined?

Reply:
I'll double-check and add `|undefined` if any can be null.  IIRC they should be all defined.